plugin/pkg/admission/podpreset/admission.go

```diff
@@ -38,6 +38,7 @@ import (
 
 const (
 	annotationPrefix = "podpreset.admission.kubernetes.io"
+	exclusionSuffix  = "PodPresetOptOut"
```

Comment:
typically, a pod will have an annotation that says:

"podpreset.admission.kubernetes.io/{pod-preset-name}" = {pod-preset-resource-version}" that mutated pod spec.

ideally, we would have made that annotation two keys rather than encoding it all in one.

i would be in favor of doing just that actually.

"podpreset.alpha.admission.kubernetes.io/name" - name of pod preset that mutated pod spec.
"podpreset.alpha.admission.kubernetes.io/resource-version" - resource version of pod preset that mutated pod spec.
"podpreset.alpha.admission.kubernetes.io/opt-out" - if true, opt-out of applying pod preset.

@pmorie - thoughts?  i don't see why we cannot change how the annotation is applied here today given alpha nature of API.

https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/podpreset/admission.go#L185